Elemental/Components/AeTable/AePaginatedTableMerged.razor

```diff
@@ -0,0 +1,125 @@
+        </AeTableRow>
+    }
+
+    <div class="ae table-rows">
+        @for (var itemIndex = 0; itemIndex < PageItems.Count; itemIndex++)
+        {
+            var item = getPageItem(itemIndex);
+            var currentIndex = itemIndex;
+            var idx = itemIndex + Page * PageSize;
+            <AeTableRow @key=idx GridStyle="@_gridStyle"
+                    tabindex="0"
+                    @onclick="(() => OnRowClick.InvokeAsync(getPageItem(currentIndex)))"
+                    @onfocus="(() => OnRowFocus.InvokeAsync(getPageItem(currentIndex)))">
+                @foreach (var (accessor, index) in Accessors.Select((a, i) => (a, i)))
+                {
+                    if (Renderers?[index] is null)
+                    {
+                        <AeTableCell>
+                            @accessor?.Invoke(item)
+                        </AeTableCell>
+                    }
+                    else
+                    {
+                        <AeTableCell>
+                            @Renderers[index](item)
+                        </AeTableCell>
+                    }
+                }
+            </AeTableRow>
+
+        }
+</div>
+</div>
+
+<AePagination Pageable="@this" OnPageClick="OnPageClick">
+	<LeftIcon>
+		@LeftIcon
+	</LeftIcon>
+	<RightIcon>
+		@RightIcon
+	</RightIcon>
+</AePagination> 
+
+@code {
+
+    [Parameter] public int PageSize { get; set; } = 5;
+    [Parameter] public int Page { get; set; } = 0;
+
+	[Parameter] public EventCallback<int> OnPageClick { get; set; }
+
+	[Parameter] public RenderFragment LeftIcon { get; set; }
+	[Parameter] public RenderFragment RightIcon { get; set; }
+
+    private List<T> PageItems;
+    private int? currentPage;
+
+    private T getPageItem(int index)
+    {
+        return PageItems[index];
+    }
+
+    public int Total { get; set; }
+
+    private void LoadPageItems(bool forceLoad = false)
+    {
+        if (forceLoad || Page != currentPage)
+        {
+            PageItems = Rows.Skip(Page * PageSize).Take(PageSize).ToList();
+            currentPage = Page;
+        }
+    }
+
+    private string _className => $"{_tableClass} paginated";
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
+        LoadPageItems();
+    }
+
+    protected override async Task OnParametersSetAsync()
+    {
+        Total = Rows.Count();
+        int PageCount = (int)Math.Ceiling(Total / (decimal)Math.Max(1, PageSize));
+        Page = (Page > PageCount - 1) ? PageCount - 1 : Page;
+        LoadPageItems(true);
+    }
+
+    protected override RenderFragment DefaultHeader(string header, int index)
+    {
+        return
+    @<AeTypography class="table-cell" @onclick="() => SortTable(index)">
+        @header
+        <span class="ae table-sort"><i class="@GetSortIconClass(index)"></i></span>
+    </AeTypography>
+    ;
+    }
+
+    private void SortTable(int index)
+    {
+        ToggleSortOnColumn(index);
+        LoadPageItems(true);
+    }
```

Comment:
This is all duplicate from `AePaginatedTable`. If we're unable to re-use the `AePaginatedTable` component we should lift this up to a parent class